Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15-buster
+FROM golang:1.17-buster
```

Comment:
As I wrote in the issue, the version of the linux distro needs to be pinned as well. We had cases at work where things stopped working from one build to the next because of changes in alpine. I would like to avoid these "its broken but I didn't touch anything" moments.

Applies to all the tooling PRs probably.

Reply:
Changed 👍